jekyll/_posts/2021-01-19-community-survey-results-2020.md

```diff
@@ -0,0 +1,221 @@
+---
+title: "Nim Community Survey 2020 Results"
+author: The Nim Team
+---
+
+Nim community survey 2020 has been open for one month, and we have received 769 responses, which we think it is large enough sample to draw the conclusions about our users and their habits.
```

Comment:
This reads wrong.

Also, would it be worth it to compare the response rate to our previous surveys?

Reply:
It is lower than last year, but last year had the survey open for 50 days, this year 30. It is comparable to 2018.
(I didn't want to write that in the article)

What reads wrong? How to improve it?

Reply:
> which we think it is large enough sample

This just reads to me like there is a better way to say it.